inference-engine/thirdparty/movidius/mvnc/src/mvnc_api.c

```diff
@@ -749,12 +750,6 @@ ncStatus_t ncDeviceOpen(struct ncDeviceHandle_t **deviceHandlePtr,
     }
 
     GLOBAL_LOCK();
-    int error = pthread_mutex_lock(&deviceOpenMutex);
```

Comment:
Also, you should remove the usages of this mutex at the line 1051, 1060 and so on